api/proto/teleport/legacy/types/types.proto

```diff
@@ -972,6 +972,10 @@ message IdentityCenterPermissionSet {
 
   // Name is the human-readable name of the Permission Set.
   string Name = 2 [(gogoproto.jsontag) = "name,omitempty"];
+
+  // AssignmentID is the ID of the Teelport Account Assignment resource that
+  // represents this permission being assigned on the enclosing Account.
+  string AssignmentID = 3 [(gogoproto.jsontag) = "assignment_name,omitempty"];
```

Comment:
My bad that I didn't notice this on the main PR but it looks weird that the field name is `ID` while the json tag is `name`.